### ISSUE TITLE
Make Proto-Token And Actor Art Align

### ISSUE THREAD

**Issue by H-O-W-L-45** [drewg13/foundryvtt-scum-and-villainy#82]
This is a simple one, I'm not sure what determines it, but. In other game systems (like Blades in the Dark or Delta Green), whenever you make an actor and assign their actor art, it also sets the prototype token to the same file.

In S&V however, this isn't the case; you have to manually set the prototype token art to the same as the actor art. While this is a minor complaint, it _is_ a little bit styming in-session, since the nature of S&V means you're often making NPC actors very quickly on the fly. So, rather than being able to very quickly create an actor and click their token icon once to have them on the canvas, I have to do it twice. Minor, but it slows a process that could be quicker-- and if you use the Drag Upload module, you can't just drag-upload tokens to make actors, since it also only sets their actor art, not their proto-token. 

Either enabling this by default, or making it an option in the system configuration, would be greatly appreciated.

**Comment by drewg13** [comment]
Interesting, the way I've played in the past, I've always started with actor art that's in a full-length portrait style and then modified it using Tokenizer to make related, but separate, token art.  I almost never placed token art in a scene where players could see them, rather I kept them in the padding and invisible in a group, so I could click and speak as them.  So token art was mostly irrelevant to me.  My workflow was completely different than yours.  I can see if that's something easy to add as a setting.
